warehouse/docker-compose.yml

```
@@ -0,0 +1,63 @@
+services:
+  trino:
+    ports:
+      - "8080:8080"
```

```suggestion
      - "${SQLMESH_TRINO_PORT:-8080}:8080"
```

Let's make this configurable (just in case the caller needs to change the port setting)